src/components/ha-form/ha-form-grid.ts

```diff
@@ -28,6 +28,9 @@ export class HaFormGrid extends LitElement implements HaFormElement {
 
   protected firstUpdated() {
     this.setAttribute("own-margin", "");
+    if (this.schema.columns) {
```

Comment:
Shouldn't this happen on every update of the schema? and unset if schema doesn't contain `columns`.